sormas-backend/src/test/java/de/symeda/sormas/backend/person/PersonFacadeEjbTest.java

```diff
@@ -926,8 +928,22 @@ public void testMergePersonsAndRemoveDuplication() {
 		otherPerson.setPhone("+496211218491");
 		leadPerson.setEmailAddress("lead@hotmail.com");
 		otherPerson.setEmailAddress("other@yahoo.com");
+
+		TestDataCreator.RDCFEntities rdcf1 = creator.createRDCFEntities("Region", "District", "Community", "Facility");
+		TestDataCreator.RDCFEntities rdcf2 = creator.createRDCFEntities("Region2", "District2", "Community2", "Facility2");
+
+		LocationDto leadPersonAddress = leadPerson.getAddress();
+		leadPersonAddress.setRegion(new RegionReferenceDto(rdcf1.region.getUuid()));
+		leadPersonAddress.setDistrict(new DistrictReferenceDto(rdcf1.district.getUuid()));
+
+		LocationDto otherPersonAddress = otherPerson.getAddress();
+		otherPersonAddress.setRegion(new RegionReferenceDto(rdcf2.region.getUuid()));
+		otherPersonAddress.setDistrict(new DistrictReferenceDto(rdcf2.district.getUuid()));
+		otherPersonAddress.setCommunity(new CommunityReferenceDto(rdcf2.community.getUuid()));
+
 		leadPerson.setAddresses(Collections.singletonList(LocationDto.build()));
-		otherPerson.setAddresses(Collections.singletonList(LocationDto.build()));
```

Comment:
the test was modified in such a way that the old asserts cover it - but i would extend the asserts as now you have more information in the addresses

Reply:
asserts extended and added more tests for different situations